pandas/tests/extension/decimal/array.py

```diff
@@ -142,8 +143,8 @@ def astype(self, dtype, copy=True):
         return super().astype(dtype, copy=copy)
 
     def __setitem__(self, key, value):
-        if pd.api.types.is_list_like(value):
-            if pd.api.types.is_scalar(key):
+        if is_list_like(value):
```

Comment:
I think the `is_list_like` import should also change to be imported from `pandas.api.types`